### PR TITLE
add ansible-operator package

### PR DESCRIPTION
**Pull request by kranurag7**
CI will fail for now given it's dependent on https://github.com/wolfi-dev/os/pull/33634

### CODE REVIEW

**Comment by maxgio92** [comment]
https://github.com/wolfi-dev/os/pull/33634 has been merged, let's wait for the package to be published

**Comment by maxgio92** [comment]
@kranurag7 the ansible-core package is now available, you can make it ready to review and test when you want

**Review by maxgio92 — APPROVED** [review]

File: ansible-operator.yaml [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+  copyright:
+    - license: Apache-2.0
+  dependencies:
+    runtime:
+      - py3-ansible-core
+      - py3-ansible-runner-http
+      - py3-kubernetes
+      - py3-pip
+      - py3-pipenv
+      - python3
```
Comment (EyeCantCU) [inline ansible-operator.yaml:15]: It only supports latest Python?

Comment (EyeCantCU) [reply ansible-operator.yaml:15]: Ah wait, we can still drop in whatever as other versions are provided... Don't mind me

**Review by EyeCantCU — APPROVED** [review]

**Pull request merged**